CARET_trace/test/test_trace_node.cpp

```diff
@@ -207,27 +252,49 @@ TEST_F(CaretTraceNodeTest, TestMeasureTransition)
 {
   init_context();
 
-  {  // subscribe start message: MEASURE -> PREPARE
+  {  // subscribe start message with lttng session: MEASURE -> PREPARE
```

Comment:
Minor fix of comment. (Let's change MEASURE to RECORD on this occasion)
```suggestion
  {  // subscribe start message with lttng session: RECORD -> RECORD
```